src/api/java/CMakeLists.txt

```diff
@@ -49,7 +57,7 @@ add_custom_target(
   ALL
   COMMAND
   # generate jni header files
-  ${Java_JAVAC_EXECUTABLE} -h jni ${java_files} -d jni
+  ${Java_JAVAC_EXECUTABLE} -h jni ${JAVA_FILES} -d jni
   COMMENT "Generate jni headers"
   VERBATIM
 )
```

Comment:
We should be able to do a `add_custom_command` here too in order to avoid unnecessary rebuilds of the JNI headers. They should only be rebuilt if `JAVA_FILES` changed.

Reply:
Will do

Reply:
Done